src/read.js

```diff
@@ -227,11 +224,7 @@ class Entry {
 }
 
 function getBigInt64(view, position, littleEndian = false) {
-  if ('getBigInt64' in DataView.prototype) {
```

Comment:
I remove this because I don't think the native BigInt, even if available, will be interoperable with JSBI. I'm assuming that the later fallback is correct and only changing BigInt usage to JSBI.